docs/data-model/zson.md

```diff
@@ -511,22 +506,21 @@ the defines their type.
 
 <value> = <any> | <any> <val-typedef> | <any> <decorators>
 
-<val-typedef> = "(" "=" <type-name> ")"
+<val-typedef> = "(" "=" <name> ")"
 
 <decorators> = "(" <type> ")" | <decorators> "(" <type> ")"
 
-<any> = <primitive> | <record> | <array> | <set> |
-            <union> | <enum> | <map> | <type-val>
+<any> = <primitive> | <record> | <array> | <set> | <enum> | <map> | <type-val>
```

Comment:
Maybe shuffle order here in the grammar to match the order these are described in the text?
```suggestion
<any> = <primitive> | <type-val> | <record> | <array> | <set> | <map> | <enum>
```